components/rhs_comment/rhs_comment.jsx

```diff
@@ -200,16 +200,17 @@ export default class RhsComment extends React.Component {
         let profilePicture;
         let visibleMessage;
 
-        let userProfile = (
-            <UserProfile
-                userId={post.user_id}
-                isBusy={this.props.isBusy}
-                isRHS={true}
-                hasMention={true}
-            />
-        );
-
+        let userProfile = null;
         if (!isConsecutivePost) {
+            userProfile = (
```

Comment:
Worth pushing this to the bottom of the block and only assigning it when `null`? Saves an extra render for all of the (admittedly infrequent) cases below that override.

Reply:
I would save it save an extra assignation, not a whole extra render. I don't mind to put this at the bottom as an else, but I think the save here will be super small.

Reply:
My mistake: you are correct, no extra render. 0/5 on whether to move it.

Reply:
I think I going to keep it at the top, because is more clear for me, and more similar to the previous approach.